frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideInBranchBelowAction.java

```diff
@@ -85,13 +76,13 @@ public void update(AnActionEvent anActionEvent) {
   @UIEffect
   public void actionPerformed(AnActionEvent anActionEvent) {
     var project = getProject(anActionEvent);
-    var selectedVcsRepository = getSelectedGitRepository(anActionEvent).getOrNull();
```

Comment:
While you're here... pls rename of couple other remaining `selectedVcsRepository` occurrences across the project